examples/selenium/maven-project/TaurusScriptV2.yml

```diff
@@ -0,0 +1,17 @@
+---
+# Requires Taurus >= 1.6.9
+execution:
+- executor: selenium
+  ramp-up: 1m
+  hold-for: 30s
+  scenario: simple-junit-test
+
+scenarios:
+  simple-junit-test:
+      script: ./src/test/java
+      additional-classpath:
+      - target/Selenium-1.0-SNAPSHOT.jar
+
+services:
+ - module: shellexec
+   prepare: mvn -Dmaven.test.skip=true clean package
```

Comment:
Not sure why, but this command hangs Taurus when launched on a freshly cloned project.

If i run `mvn package` manually in terminal before launching bzt - this command succeeds and Taurus then works as expected.